From 4fca9a1203916489bd12f621610c77e9a646a2d9 Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Fri, 2 May 2014 14:36:56 -0400
Subject: [misc] mei: replace stray pr_debug with dev_dbg

Message-id: <1399041470-21330-24-git-send-email-prarit@redhat.com>
Patchwork-id: 79310
O-Subject: [RHEL7.1 PATCH BZ 1088953 23/77] mei: replace stray pr_debug with dev_dbg
Bugzilla: 1088953
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Dean Nelson <dnelson@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1088953

commit c4e87b525936da188add18d53a0fe681c3c977ce
Author: Alexander Usyskin <alexander.usyskin@intel.com>
Date:   Mon Oct 21 22:05:42 2013 +0300

    mei: replace stray pr_debug with dev_dbg

    Driver better use dev_dbg, not pr_debug.

    Signed-off-by: Alexander Usyskin <alexander.usyskin@intel.com>
    Signed-off-by: Tomas Winkler <tomas.winkler@intel.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/misc/mei/pci-me.c b/drivers/misc/mei/pci-me.c
index 862f90b..0d2a5de 100644
--- a/drivers/misc/mei/pci-me.c
+++ b/drivers/misc/mei/pci-me.c
@@ -221,7 +221,7 @@ static int mei_me_probe(struct pci_dev *pdev, const struct pci_device_id *ent)
 
  mutex_unlock(&mei_mutex);
 
- pr_debug("initialization successful.\n");
+ dev_dbg(&pdev->dev, "initialization successful.\n");
 
  return 0;
 
-- 
1.7.1